front/lib/api/assistant/agent.ts

```diff
@@ -422,6 +428,7 @@ export async function getNextAction(
   config.MODEL.function_call = forcedActionName ?? "auto";
   config.MODEL.provider_id = model.providerId;
   config.MODEL.model_id = model.modelId;
+  config.MODEL.temperature = 0.2;
```

Comment:
Meh

Reply:
Should be the assistant config for now no?

Reply:
So we can play with it easily. Maybe in the future we’ll add a config

Reply:
> Should be the assistant config for now no?

Mmh fair. Although one could also argue we might want a different setting for scheduling vs generation. But for now let's use assistant's config indeed. Making a PR 👍